### ISSUE TITLE
Incorrect output values in issue_metrics.json

### ISSUE THREAD

**Issue by eichisanden** [github/issue-metrics#161]
Incorrect output values for `average_time_to_first_response` and `average_time_to_close` in issue_metrics.json
The actual values in issue_metrics.json are shown below, but the timedelta map values have been output.

```json
{
    "average_time_to_first_response": "{'avg': datetime.timedelta(seconds=50124), 'med': datetime.timedelta(seconds=10122), '90p': datetime.timedelta(days=1, seconds=15064)}",
    "average_time_to_close": "{'avg': datetime.timedelta(seconds=69549), 'med': datetime.timedelta(seconds=19616), '90p': datetime.timedelta(days=2, seconds=78098)}",
    "average_time_to_answer": "None",
    "average_time_in_labels": {
```

https://github.com/github/issue-metrics/blob/9291773ac94d1e6e9afa852d792e92230f882582/json_writer.py#L85C22-L85C22

In the above code, the variable `average_time_to_first_response` used to be of type timedelta, but it has changed to type map, so  need to specify the element to get.

**Comment by eichisanden** [comment]
fixed in #163

**Comment by eichisanden** [comment]
I also added #164 support for median and 90th percentile.
Please merge the ones you like.
